api/tests/functional-tests/backend/metrics/test_detection.py

```diff
@@ -160,6 +182,8 @@ def test__compute_curves(db: Session):
                 dataset_name="test_dataset",
                 pd_datum_id=527,
```

Comment:
should we remove `datum_id`s since they can't be used externally?

Reply:
RankedPairs require the actual id b/c they are also used in calculating AP. They are internal storage and dont reflect what is returned to the user.

The datum_id position in the returned metric is now populated by the datum UID instead.

Reply:
The returned tuple is (dataset_name, datum_uid, geojson)

Reply:
aw that's right, thanks for clarifying